api-docs/modern/modern/src/grid/GridDragZone.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @private
```

Comment:
Is this an orphan?  Something is missing here?

Reply:
Yes, We have added @class and @extend in docblock.